modules/effects/src/effect_sources.ts

```diff
@@ -14,18 +14,25 @@ import {
   reportInvalidActions,
   EffectNotification,
 } from './effect_notification';
-import { mergeEffects } from './effects_resolver';
+import { mergeEffects, EffectsErrorHandler } from './effects_resolver';
 import {
   onIdentifyEffectsKey,
   onRunEffectsKey,
   OnRunEffects,
   onInitEffects,
 } from './lifecycle_hooks';
+import { EFFECTS_ERROR_HANDLER } from './tokens';
 import { getSourceForInstance } from './utils';
 
 @Injectable()
 export class EffectSources extends Subject<any> {
-  constructor(private errorHandler: ErrorHandler, private store: Store<any>) {
+  constructor(
+    private errorHandler: ErrorHandler,
+    private store: Store<any>,
+    @Optional()
+    @Inject(EFFECTS_ERROR_HANDLER)
```

Comment:
[Optional]: just a thought
I wonder if instead of making it `@Optional()` here, maybe it's worth providing it in the EffectsRootModule?
WDYT? @timdeschryver WDYT as well?

Reply:
Hmm I'm on the fence as it doesn't really change much functionality, the choice may be more based on future API changes. In some sense it would be tidier as most of the other symbols are provided there too. Tell you what I'll make the change in a separate commit that I can easily drop or squash based on feedback from there.

Reply:
OK new commit added to PR with just this change. I also had to rebase hence the noise on the PR pushes.

Reply:
Sorry for the late response, but I agree here with @alex-okrushko 
But I see it's already resolved 👍👍